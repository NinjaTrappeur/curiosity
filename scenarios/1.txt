# This shows that the first user can verify an email address, but not the
# second user.

reset
as alice
user create alice secret alice@example.com --accept-tos
user create bob secret bob@example.com --accept-tos
user do set-email-addr-as-verified USER-2  # Will succeed.
user do set-email-addr-as-verified USER-2  # This is already done.

reset
as bob
user create alice secret alice@example.com --accept-tos
user create bob secret bob@example.com --accept-tos
user do set-email-addr-as-verified USER-2  # Will fail: missing right.
