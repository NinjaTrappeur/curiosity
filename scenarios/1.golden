4: reset
Resetting to the empty state.
5: as alice
Modifiying default user.
6: user create alice secret alice@example.com --accept-tos
User created: USER-1
7: user create bob secret bob@example.com --accept-tos
User created: USER-2
8: user do set-email-addr-as-verified USER-2
User successfully updated.
9: user do set-email-addr-as-verified USER-2
EmailAddrAlreadyVerified
11: reset
Resetting to the empty state.
12: as bob
Modifiying default user.
13: user create alice secret alice@example.com --accept-tos
User created: USER-1
14: user create bob secret bob@example.com --accept-tos
User created: USER-2
15: user do set-email-addr-as-verified USER-2
MissingRight CanVerifyEmailAddr
